angular-ngrx-scss/src/app/shared/components/filter-dropdown/filter-dropdown.component.html

```diff
@@ -1,6 +1,11 @@
 <ul class="filter-dropdown-container">
   <div>
-    <button role="button" class="control-dropdown" (click)="toggleMenu()">
+    <button
+      role="button"
+      class="control-dropdown"
+      [ngClass]="{ isRepo: isRepo }"
```

Comment:
```suggestion
      [ngClass]="{ 'control-dropdown--repo': isRepo }"
```

For reasoning, see the other comment in filter-dropdown.component.scss 😄